### PR TITLE
feat: Add Allied Universal spider

### LINKED ISSUE #1693: Allied Universal
https://www.aus.com/offices

### PR DESCRIPTION
**Pull request by rubyruins**
Fixes #1693.

### CODE REVIEW

File: locations/spiders/allied_universal.py [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    "TB": "Tabasco",
+    "TL": "Tlaxcala",
+    "TM": "Tamaulipas",
+    "VE": "Veracruz",
+    "YU": "Yucatan",
+    "ZA": "Zacateca",
+}
+
+
+class TemplateSpider(scrapy.Spider):
```
Comment (Cj-Malone) [inline locations/spiders/allied_universal.py:98]:
```suggestion
class AlliedUniversalSpider(scrapy.Spider):
```

File: locations/spiders/allied_universal.py [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+        country = (
+            location.xpath('.//div[@class="Country"]/text()').extract_first().strip()
+        )
+        if not country:
+            country = "USA"
+            if state in USA_STATES:
+                state = USA_STATES[state]
+        elif country == "Mexico":
+            if state in MEXICO_STATES:
+                state = MEXICO_STATES.get(state)
```
Comment (Cj-Malone) [inline locations/spiders/allied_universal.py:134]: I don't know if I'd bother with the state code -> state text, but I'll let @iandees decide.

File: locations/spiders/allied_universal.py [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+        phone = (
+            location.xpath('.//div[@class="PhoneNum"]/text()')
+            .extract_first()
+            .replace(".", "-")
+            .strip()
+        )
+        addr_full = street_address + " " + state + " " + postcode
```
Comment (Cj-Malone) [inline locations/spiders/allied_universal.py:141]:
```suggestion
        addr_full = street_address + ", " + state + ", " + postcode
```

**Comment by Cj-Malone** [comment]
Looks good, would it also be possible to get the website url when it's available?

**Comment by rubyruins** [comment]
> Looks good, would it also be possible to get the website url when it's available?

@Cj-Malone, sure, I can include it - however, every office does not have a separate website, and many of them point to the common website for that state, hence the website fields will be duplicated. Will that be okay?

**Comment by Cj-Malone** [comment]
Yeah I'd put the website in. It's not unique to the POI, but it's more specific than nothing, and it does list the addresses on there.

It's not idea, but it's better than nothing.

**Comment by rubyruins** [comment]
Sounds good, @Cj-Malone. I've added the website field now.

**Review by Cj-Malone — APPROVED** [review]
LGTM

**Pull request merged**